app/myapp/views.py

```diff
@@ -553,3 +559,42 @@ def is_deadline_over():
 
 def deadlineover(request):
     return render(request, 'myapp/deadlineover.html')
+
+
+@login_required()
+def closeFriends(request):
+    if is_deadline_over():
+        return deadlineover(request)
+
+    u = request.user
+    UsrObj = Student(name=u.student.name, department=u.student.department,
+        DP=u.student.DP, phone=u.student.phone, email=u.student.email,
+        oneliner=u.student.oneliner, future=u.student.future, genPic1=u.student.genPic1, genPic2=u.student.genPic2, closeFriends=u.student.closeFriends, closeFriendsPic=u.student.closeFriendsPic)
+    users_all = User.objects.filter(is_superuser=False).order_by('username')
+    users_json = []
+    for user in users_all:
+        users_json.append({"id": user.username, "user": user.student.name + " (" + user.username + ")"})
```

Comment:
The issue is on this line, when testing on my machine.
`user.student` does not exist.
`user.username` places no issue